v3/plugins/redhat/vscode-xml/0.11.0/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-xml
+version: 0.11.0
+type: VS Code extension
+displayName: XML
+title: XML Language Support by Red Hat
+description: This VS Code extension provides support for creating and editing XML documents, based on the LSP4XML language server, running with Java.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-xml
+category: Language
+firstPublicationDate: "2020-04-15"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-java:11-f76ca45"
+      name: vscode-xml
+      memoryLimit: "768Mi"
+  extensions:
+    - https://github.com/redhat-developer/vscode-xml/releases/download/0.11.0/redhat.vscode-xml-0.11.0.vsix
```

Comment:
dunno if this is interesting/better for you guys but the vsix is also available on open-vsx.org: https://open-vsx.org/api/redhat/vscode-xml/0.11.0/file/redhat.vscode-xml-0.11.0.vsix